AUTHORS

```diff
@@ -1,5 +1,2 @@
-Martin Aigner <martin@maigner.net>
-Christian Barthel <cbarthel@cs.uni-salzburg.at>
-Christoph Kirsch <ck@cs.uni-salzburg.at>
-Michael Lippautz <michael.lippautz@gmail.com>
-Simone Oblasser <simone.oblasser@cs.uni-salzburg.at>
\ No newline at end of file
+Tobias Seiler <tobias.seiler@stud.sbg.ac.at>
+Johannes Vollmer <johannes.vollmer@stud.sbg.ac.at>
```

Comment:
see comment on the AUTORS file from Assignment 0. Remember that your repository is public!